.github/workflows/update.yml

```diff
@@ -0,0 +1,33 @@
+name: Update
+on:
+  pull_request:
+    types: closed
+  workflow_dispatch:
+jobs:
+  build:
+    name: Deploy docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Checkout LFS objects
+        run: git lfs pull
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 'pypy-2.7'
```

Comment:
Should switch to py3

Reply:
@AcylSilane said that at the moment we can't go to py3.  We need to update one of the libraries that only work with py2. 
We need to create a separate task about migrating the documentation to py3 and updating all the packages to the latest versions.

Reply:
Why are we using pypy at all?

Reply:
Which library is that?

I was able to run the docs with py3 a while ago, it shouldn't be all that difficult, but yes - let's have a separate task to track it.

Reply:
https://exabyte.slack.com/archives/CFGN3K05C/p1638552607082400?thread_ts=1638551239.080400&cid=CFGN3K05C

I will create the task to update all packages and migrate the environment to py3